pynicotine/gtkgui/search.py

```diff
@@ -300,9 +300,10 @@ class Search:
         ("archive", FileTypes.ARCHIVE)
     )
     FILTER_PRESETS = {
-        "filterbr": ("0", "128", "160", "192", "256", "320"),
-        "filtersize": (">10MiB", "<10MiB", "<5MiB", "<1MiB", ">0"),
-        "filtertype": ("audio", "image", "video", "text", "archive", "!executable", "audio|image|text")
+        "filterbr": ("!0", "128 <224", ">190 <=320", "=320 =1411", "320"),
+        "filtersize": (">50MiB", ">20MiB <=50MiB", ">10MiB <=20MiB", ">5MiB <=10MiB", "<=5MiB"),
+        "filtertype": ("audio", "image", "video", "text", "archive", "!executable", "audio image text"),
+        "filterlength": (">12:00", ">8:00 <=12:00", ">5:00 <=8:00", "!0 <=5:00", "=0")
```

Comment:
Not sure if it's worth listing the `=0` filter by default, now that we have the generic file type filters.

Regarding `!0 <=5:00` wouldn't it be more intuitive if simply `<=5:00` worked, and files without a duration (or 0) were excluded automatically? Right now it also lists non-audio files, which feels wrong.

Reply:
Agreed, what was offered here is a solution to a problem that shouldn't exist in the first place.